addons/mrp_subcontracting/views/stock_picking_views.xml

```diff
@@ -10,9 +10,17 @@
                 <button name="action_record_components" class="oe_highlight" attrs="{'invisible': [('display_action_record_components', '=', False)]}" string="Record components" type="object" data-hotkey="shift+x"/>
             </xpath>
             <xpath expr="//field[@name='move_ids_without_package']//tree//button[@name='action_show_details']" position="after">
+                <field name="show_serial_mass_receipt" invisible="1"/>
+                <button name="action_serial_mass_receipt" type="object" icon="fa-list" width="0.1"
+                    string="Mass receipt"
+                    attrs="{'invisible': [('show_serial_mass_receipt', '=', False)]}" options='{"warn": true}'/>
                 <field name="show_subcontracting_details_visible" invisible="1"/>
                 <button name="action_show_subcontract_details" string="Register components for subcontracted product" type="object" icon="fa-sitemap"
                     width="0.1" attrs="{'invisible': [('show_subcontracting_details_visible', '=', False)]}"/>
+                <field name="is_subcontract" invisible="1"/>
+            </xpath>
+            <xpath expr="//field[@name='move_ids_without_package']//tree//button[@name='action_assign_serial']" position="attributes">
+                <attribute name="attrs">{'invisible': ['|', '|', ('display_assign_serial', '=', False), ('show_operations', '=', False), ('is_subcontract', '=', True)]}</attribute>
```

Comment:
I think something isn't quite correct here? 

![Odoo-WH-IN-00035](https://user-images.githubusercontent.com/57942143/131341091-2128aeb2-f26f-43b0-a1d7-5a61e39b96ae.png)

First burger:
![Screenshot from 2021-08-30 14-46-42](https://user-images.githubusercontent.com/57942143/131341208-d4959108-ba73-4814-b34d-2b13a4e9ee7f.png)

Second burger:
![Screenshot from 2021-08-30 14-47-06](https://user-images.githubusercontent.com/57942143/131341248-4775b15c-50ad-414b-a91f-227e2161f4e9.png)

I didn't look closely as to why this is happening, but the first burger is showing `move_line_ids` while the second burger is showing `move_line_nosuggest_ids`. I have Storage Locations active if that helps.

Reply:
this is related to show_reserved (Pre-fill Detailed Operations), standard process

Reply:
I think it's confusing to have both show what looks to be the same window, but with different info in it. There also seems to be some weird behavior where sometimes the first burger shows the "Detailed Operations" window and other times it shows the "Subcontract" window, specifically if you delete one of the `move_line_ids` lines, close the window and reopen. This is maybe something to consider fixing as a follow up task.